app/javascript/courses/review/CoursesReview__Root.res

```diff
@@ -29,6 +29,7 @@ type state = {
   targetsLoaded: loading,
   levelsLoaded: loading,
   coachesLoaded: loading,
+  defaultSortDirection: Filter.sortDirection,
```

Comment:
You are breaking the single source of truth with an addition of a state variable, which isn't required in this case. The sort direction is already available via the route.